docs/best-practices.md

```diff
@@ -0,0 +1,51 @@
+# Best practices for writing actors
```

Comment:
I miss some information like "review existing actors/models". We already have many data gathering actors (i.e system facts) that a new actor can use without the necessity to inspecting the system again.

Reply:
For this purpose we would document everything in the code and generate overview/documentation automatically with every merge.

Reply:
Thanks, good idea. I have added section suggesting to check out existing actors.